pkg/yurthub/sharecache/cache.go

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
It is hard to understand `sharecahe` in yurthub, because of there is a `cachemanager` directory. so how about rename `sharecache` to `listwatchmerger`?

Reply:
How about using a multiplexer?

Reply:
I think it's more understandable to rename `multiplexer`.